tests/functions/test_encode_categorical.py

```diff
@@ -62,10 +62,10 @@ def test_encode_categorical_missing_columns(df):
 @settings(deadline=None)
 def test_encode_categorical_invalid_input(df):
     """
-    Raise ValueError for wrong input type
+    Raise KeyError for wrong input type
     for `column_names`.
     """
-    with pytest.raises(ValueError):
+    with pytest.raises(KeyError):
         df.encode_categorical(1)
```

Comment:
Not quite sure I understand the point of this test anymore, since if there was a datafame with column name (1, the int), it would work fine. So essentially this test is the same as the `test_encode_categorical_missing_column()` test above.